lib/scala/syntax/definition/src/main/scala/org/enso/syntax/text/AST.scala

```diff
@@ -15,24 +13,3 @@ import cats.Monoid
 case class Location(start: Int, end: Int) {
   def length: Int = end - start
 }
-
-object Location {
-  implicit val optionSpanMonoid: Monoid[Option[Location]] =
```

Comment:
Looks like this `Monoid` is no longer needed by anything. Deleting the code allows us to remove dependency on `cats`: 1432e138dabf1aa8d10c3bafc75d6ad9d85509f5 - the less dependencies our parser into `IR` is going to have, the better.